internal/pkg/deploy/deploy.go

```diff
@@ -163,17 +172,29 @@ func (s *Store) IsDeployed(appName string, envName string, svcName string) (bool
 	return false, nil
 }
 
+// // getServiceName gets the ECS service name given a specific ARN.
+// // For example: arn:aws:ecs:us-west-2:123456789012:service/my-app-test-Cluster-cdgG2k6XIBtM/my-app-test-my-svc-Service-WLYA7MGACV1F
+// // returns my-service
+// func (s *Store) getServiceName(svcARN string) (string, error) {
```

Comment:
Use ECS API to get all tags for ECS service allows us to get a "safer" service name. However, it requires calling ECS get tags API.

Reply:
I think we can modify `rgClient.GetResourcesByTags` to return to us:
```go
[{
    arn: string,
    tags: map[string]string,
}]
```
and tap into tags[ServiceTagKey] to get the name of the svc